lib/unit_tests.html

```diff
@@ -11,8 +11,8 @@
     <title>Nessie Javascript SDK Tests</title>
     
     <!-- QUnit Required Files -->
-    <link rel="stylesheet" href="//code.jquery.com/qunit/qunit-1.20.0.css">
-    <script src="//code.jquery.com/qunit/qunit-1.20.0.js"></script>
+    <link rel="stylesheet" href="http://code.jquery.com/qunit/qunit-1.20.0.css">
+    <script src="http://code.jquery.com/qunit/qunit-1.20.0.js"></script>
```

Comment:
I was running the tests via the browser and couldn't load the files until I tried using the absolute path with `http` prefix:

![image](https://cloud.githubusercontent.com/assets/6519458/18533859/3836249c-7ab4-11e6-9eeb-58ce9e07d82d.png)

How are you running the tests?